lib/queue-integration.ts

```
@@ -0,0 +1,370 @@
+     * Queue workout completion notification
+     */
+    static async queueWorkoutCompletionNotification(
+        userId: string,
+        workoutName: string,
+        completedExercises: number,
+        totalExercises: number
+    ) {
+        const title = "Workout Completed! 🎉";
+        const message = `Great job! You completed ${completedExercises}/${totalExercises} exercises in "${workoutName}".`;
+
+        return this.queueNotification(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)